### DISCUSSION TITLE
Add a option to remove the random button

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#19125] [Ideas] [answered]
The "random" button is right between the "mods" and the "options" button which are both commonly used, so there are many chances for a misclick. I can always press <kbd>shift</kbd>+<kbd>F2</kbd> to go back to the previous beatmap, but this is still one of the most common inconvenience in lazer.

**Comment by peppy** [comment, marked as answer]
Sorry but this won't be a thing.

**Reply by nagi-desuuu** [reply]
There's always the option of rebinding keys, if I remember correctly, so that's that.

**Reply by developomp** [reply]
> There's always the option of rebinding keys, if I remember correctly, so that's that.

I want the button UI to be gone. Not the keyboard shortcut.
